website/content/en/docs/contribution-guidelines/releasing.md

```diff
@@ -202,6 +202,8 @@ git push -f upstream latest
 - Post to Kubernetes slack in #kubernetes-operators and #operator-sdk-dev.
 - In the [GitHub milestone][gh-milestones], bump any open issues to the following release.
 
+**Note**
+In case there are errors while building binaries for the released tag, you may have to revert the release. To do so, create a PR which reverts step [2][release_PR]. If there are changes in go modules of the underlying commit which caused the failure, it is suggested that we skip that particular version and release the next version.
```

Comment:
We could also just do this `[2](#2-create-and-merge-a-new-pr)` for the link. it's pretty short since it's just a nametag.